src/settings/settingsdialog.cpp

```diff
@@ -105,6 +109,9 @@ SettingsDialog::SettingsDialog(MainWindow *parent)
 
     ui->ocrLanguagesListWidget->addLanguages(parent->ocr()->availableLanguages());
 
+    for (int i = 1; i <= QOnlineTranslator::Zulu; ++i)
+        ui->googleLanguageComboBox->addItem(QOnlineTranslator::languageName(static_cast<QOnlineTranslator::Language>(i)), i);
```

Comment:
What if we iterate only over languages that actually have region settings using `validRegions`? This should improve UX because it hard to navigate through a lot of languages.